app/components/plan/Workout.js

```diff
@@ -4,19 +4,22 @@ import { StepsActionButton } from '../material-style'
 import FontIcon from 'material-ui/FontIcon'
 import { formatTime } from '../../utils'
 import moment from 'moment';
+import { Link } from 'react-router'
 
-export default ({ treatment, num }) => {
+export default ({ patientId, planId, treatment, num }) => {
   const completedWorkout = treatment.workouts && treatment.workouts.find(workout => {
     return moment(workout.created_at).format('MM/DD/YYYY') ===
       moment().format('MM/DD/YYYY');
   })
-  console.log(num)
+
   return (
     <div className={`row workout ${completedWorkout ? 'completed' : ''}`}>
       <div className="workout-index">#{num}</div>
       <img className="col-xs-3 workout-pic" src={treatment.exercise.img_url} />
       <div className="col-xs-3 workout-info">
-        <h2>{treatment.exercise.title}</h2>
+        <Link to={`/patients/${patientId}/plans/${planId}/treatments/${treatment.id}`}>
```

Comment:
I am not a fan of this Link. If you are a patient, I think it would be odd to have this route in your browser. I suggest `plan/treatments/:treatmentId`